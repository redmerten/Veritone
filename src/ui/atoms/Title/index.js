// @flow

import { rem } from 'polished';
import styled from 'styled-components';

const Title = styled.h1.attrs({ 'data-testid': 'atom-title' })`
  color: ${({ theme }) => theme.colors.jet};
  font-size: ${rem(20)};
  font-weight: 500;
  margin: 0;
`;

Title.displayName = 'Title';

export default Title;
