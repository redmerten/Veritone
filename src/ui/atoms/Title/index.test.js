// @flow

import React from 'react';

import Title from '.';

import { render } from 'utils/testing';

describe('atoms/Title', () => {
  it('matches its base snapshot', () => {
    const { baseElement } = render(<Title />);
    expect(baseElement).toMatchSnapshot();
  });

  it('renders Title atom', () => {
    const { getByTestId } = render(<Title />);
    expect(getByTestId('atom-title')).toBeInTheDocument();
  });
});
