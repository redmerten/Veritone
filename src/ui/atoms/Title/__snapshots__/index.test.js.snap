// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`atoms/Title matches its base snapshot 1`] = `
.c0 {
  color: #333;
  font-size: 1.25rem;
  font-weight: 500;
  margin: 0;
}

<body>
  <div>
    <h1
      class="c0"
      data-testid="atom-title"
    />
  </div>
</body>
`;
